docs/howto/connect_from_java.md

```diff
@@ -33,10 +33,8 @@ conjure-java just generates client/server interfaces, so to actually make networ
 Pass your interface as the first argument to have a client created:
 
 ```java
-RecipeBookService recipeBookService = JaxRsClient.create(
-       RecipeBookService.class,
-       UserAgent.of(Agent.of("your-server", "0.0.0")),
-       NoOpHostEventsSink.INSTANCE,
+FlightSearchService flightSearch = JaxRsClient.create(
+       FlightSearchService.class,
```

Comment:
If someone copies and pastes your new snippet, it won't compile.

https://github.com/palantir/conjure-java-runtime/blob/develop/conjure-java-jaxrs-client/src/main/java/com/palantir/conjure/java/client/jaxrs/JaxRsClient.java

Is this intentional?

Reply:
sorry, this wasn't meant for merging, I just wanted to see if I can get a rendered preview while the offline thing didn't work.